application-task-manager-ui/src/main/resources/TaskManager/Code/TaskMacroInit.xml

```diff
@@ -126,7 +126,7 @@
     <property>
       <code>require.config({
   paths: {
-    'moment-setup': new XWiki.Document(XWiki.Model.resolve('TaskManager.Code.MomentJSConfig', XWiki.EntityType.DOCUMENT)).getURL('jsx', 'minify=false'),
+    'moment-setup': new XWiki.Document(XWiki.Model.resolve('TaskManager.Code.MomentJSConfig', XWiki.EntityType.DOCUMENT)).getURL('jsx', 'minify=true'),
```

Comment:
```suggestion
    'moment-setup': new XWiki.Document(XWiki.Model.resolve('TaskManager.Code.MomentJSConfig', XWiki.EntityType.DOCUMENT)).getURL('jsx'),
```

Reply:
Require.js looks for `xwiki/bin/jsx/TaskManager/Code/MomentJSConfig.js`, that does not exist. Adding `minify=true` or any other query seems to fix the problem. Should I maybe use a time stamp instead of `minify=true`?

Reply:
Right, I forgot about this. Any query string will do. I've seen this https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemesCode/ThemeSheet.xml#L453-L456 in a couple of places, but using ``'minify=true'`` is fine too.